release-notes.md

```diff
@@ -1,3 +1,74 @@
+## 2018-10-15
+
+
+## Install
+
+To install the command line tools, please see [dcrinstaller](https://github.com/decred/decred-release/tree/master/cmd/dcrinstall).
+
+To install decrediton download, uncompress, and run [decrediton Linux](https://github.com/decred/decred-binaries/releases/download/v1.3.1/decrediton-1.3.1.tar.gz) or [decrediton OSX](https://github.com/decred/decred-binaries/releases/download/v1.3.1/decrediton-v1.3.1.dmg) or [decrediton Windows](https://github.com/decred/decred-binaries/releases/download/v1.3.1/decrediton-v1.3.1.exe).
+
+See manifest-v1.3.1.txt, and the package specific manifest files for sha256 sums and the associated .asc files to confirm those shas.
+
+See [README.md](./README.md#verifying-binaries) for more info on verifying the files.
+
+
+## Contents
+
+* [decrediton](#decrediton-v131)
+
+
+# decrediton v1.3.1
+
+This is primarily a patch release for Decrediton, but also reveals some exciting
+new functionality to the entire user base.  Previously, SPV wallet functionality
+and Politeia access were hidden behind configuration settings (which were only
+accessible by knowledgeable users).  Now Politeia is seen by default underneath
+the new "Governance" page that can be found on the left sidebar.  SPV can be
+enabled by going to the Settings page and updating the option (this will
+cause decrediton to reset to begin using the systems correctly).
+
+We are also proud to add our first round of translations.  We have added French,
+Spanish, German, Japanese, Chinese (simplified).  We've noticed some odd
+translation choices, if you notice any odd translations you can open an issue on
+[github](https://www.github.com/decred/decrediton/issues) or contact the team on
+the slack/matrix chat platforms.  Our tentative next list of languages are:
+Russian, Arabic, Malay/Tagalog.  We are also trying to formalize some of the
+translation procedures so long standing community members can offer feedback
+about mistranslated words and phrases. 
```

Comment:
Maybe we could add an emphasized "please do not request to participate in translation efforts if you're not already involved in the project in some other capacity. We're not currently looking for nor will compensate individual translators, but feel free to point out serious mistakes where you find them.".